source/common/conn_pool/conn_pool_base.h

```diff
@@ -124,6 +124,7 @@ class ActiveClient : public LinkedObject<ActiveClient>,
   Event::TimerPtr connection_duration_timer_;
   bool resources_released_{false};
   bool timed_out_{false};
+  bool allows_early_data_{false};
```

Comment:
nit: I think it might be a good idea to comment the meaning of this field.

Reply:
I removed this variable from this PR.